ci/cron/src/Main.hs

```diff
@@ -299,7 +299,7 @@ verify_signatures bash_lib tmp version_tag = do
             "if ! test -f $f.asc; then",
                 "echo $p: no signature file",
             "else",
-                "if gpg_verify $f.asc >$LOG 2>&1; then",
+                "if gpg_verify $f.asc >/dev/null; then",
```

Comment:
How much do we care about the output being quiet here? Seems useful to see why it failed when it fails.

Reply:
In my (admittedly limited) testing GPG output is not all that useful and basically wouldn't tell us more than which file failed.

Reply:
Here's an example output:
```
$ gpg --homedir $TMP --verify hello.asc
gpg: assuming signed data in 'hello'
gpg: Signature made Thu Oct  8 13:33:39 2020 CEST
gpg:                using RSA key 7694EDB3A8AD27451044EABBF00B70F12FAA72B4
gpg: BAD signature from "test signin (blabla) <a@b.c>" [ultimate]
```

Reply:
I guess I was less thinking of a signature fail and more something like a broken file path, failure to read the file, … which we cannot easily reproduce locally. But patching the output when it fails to figure out why is probably also reasonable.

Reply:
Added a commit for that. Note that `shell` takes care of printing out all of stdout and stderr for the entire shell command on non-zero exit.